yt/tests/test_load_archive.py

```diff
@@ -0,0 +1,99 @@
+import tarfile
+import time
+
+import pytest
+
+from yt.config import ytcfg
+from yt.loaders import load_archive
+from yt.sample_data.api import _download_sample_data_file, get_data_registry_table
+from yt.testing import requires_module_pytest
+from yt.utilities.exceptions import YTUnidentifiedDataType
+
+
+@pytest.fixture()
+def data_registry():
+    yield get_data_registry_table()
+
+
+@pytest.fixture()
+def tmp_data_dir(tmp_path):
+    pre_test_data_dir = ytcfg["yt", "test_data_dir"]
+    ytcfg.set("yt", "test_data_dir", str(tmp_path))
+
+    yield tmp_path
+
+    ytcfg.set("yt", "test_data_dir", pre_test_data_dir)
+
+
+# Note: ratarmount cannot currently be installed on Windows as of v0.8.1
+@requires_module_pytest("pooch")
+@requires_module_pytest("ratarmount")
```

Comment:
This decorator adds a supplement of magic over `requires_module`, it is implemented to provide more complete errors :)
```suggestion
@requires_module_pytest("pooch", "ratarmount")
```